polynote-kernel/src/main/scala/polynote/kernel/Result.scala

```diff
@@ -22,8 +22,28 @@ sealed abstract class ResultCompanion[T <: Result](msgId: Byte) {
   implicit val discriminator: Discriminator[Result, T, Byte] = Discriminator(msgId)
 }
 
-final case class Output(contentType: String, content: String) extends Result
+final case class Output(contentType: String, content: Vector[String]) extends Result
 object Output extends ResultCompanion[Output](0) {
+  def split(str: String): Vector[String] = {
```

Comment:
This is because neither the built-in `split` nor `linesWithSeparators` behave quite as I need (just a simple split, only on `\n`, with no trimming or removal of empties)